mkdocs.yml

```diff
@@ -16,6 +16,7 @@ nav:
       - Pulp Resource Manager: roles/pulp-resource-manager.md
       - Pulp Routes: roles/pulp-routes.md
       - Pulp Worker: roles/pulp-worker.md
+      - Pulp Web: roles/pulp-web.md
```

Comment:
we need a symlink at `docs/`:
https://github.com/chambridge/pulp-operator/tree/5657-build-web-server-image/docs/roles
Mkdocs only get the files inside `docs/`
https://pulp-operator.readthedocs.io/en/latest/contributing/#docs-testing

Reply:
symlink added. Thanks for the link to test appropriately. Looks good now:

![image](https://user-images.githubusercontent.com/29379759/109550422-415cff80-7a9d-11eb-8d16-7283b7f9b963.png)